test/e2e/framework.go

```diff
@@ -609,7 +609,8 @@ func (data *TestData) mutateFlowAggregatorConfigMap(ipfixCollector string) error
 	}
 	flowAggregatorConf, _ := configMap.Data[flowAggregatorConfName]
 	flowAggregatorConf = strings.Replace(flowAggregatorConf, "#externalFlowCollectorAddr: \"\"", fmt.Sprintf("externalFlowCollectorAddr: \"%s\"", ipfixCollector), 1)
-	flowAggregatorConf = strings.Replace(flowAggregatorConf, "#flowExportInterval: 60s", "flowExportInterval: 5s", 1)
+	flowAggregatorConf = strings.Replace(flowAggregatorConf, "#activeFlowRecordTimeout: 60s", "activeFlowRecordTimeout: 4s", 1)
```

Comment:
what is this change for? Maybe add a comment about this choice of values

Reply:
Added a comment.